next/app/about/leadership/OfficerCard.tsx

```diff
@@ -0,0 +1,41 @@
+import { GitHub, LinkedIn, Email } from "../../../components/common/Icons";
+
+interface OfficerCardProps {
+  item: {
+    avatar: string;
+    name: string;
+    title: string;
+    desc: string;
+    linkedin: string;
+    github: string;
+    email: string;
+  };
+}
+
+export default function OfficerCard({ item }: OfficerCardProps) {
+  let iconColor: string = "";
+
+  return (
+    <div className="mt-4">
+      <div className="w-24 h-24 mx-auto">
+        <img src={item.avatar} className="w-full h-full rounded-full" alt="" />
+      </div>
+      <div className="mt-2">
+        <h4 className="font-bold sm:text-lg text-primary-focus">{item.name}</h4>
+        <p className="font-semibold">{item.title}</p>
+        <p className="mt-2 px-2">{item.desc}</p>
+        <div className="mt-4 flex justify-center gap-4 text-gray-400">
```

Comment:
The "gray-400" is a tailwind color, and won't change with the daisyUI data theme. Is there an alternative color from the daisyUI config color schemes that work well in dark/light modes?

Reply:
'neutral' should serve this purpose